actors/power/src/types.rs

```diff
@@ -65,3 +66,29 @@ pub struct CurrentTotalPowerReturn {
     pub pledge_collateral: TokenAmount,
     pub quality_adj_power_smoothed: FilterEstimate,
 }
+
+#[derive(Serialize_tuple, Deserialize_tuple)]
+#[serde(transparent)]
+pub struct NetworkRawPowerReturn {
+    #[serde(with = "bigint_ser")]
+    pub raw_byte_power: StoragePower,
+}
+
+impl Cbor for NetworkRawPowerReturn {}
+
+#[derive(Serialize_tuple, Deserialize_tuple)]
+#[serde(transparent)]
+pub struct MinerRawPowerParams {
+    pub miner: Address,
```

Comment:
You didn't resolve this address, so it'll only work as an ID address. This raises another API design question: Addresses or actor IDs?

Reply:
Discussed offline: Actor IDs.